test/e2e/e2e_test.go

```diff
@@ -20,10 +20,17 @@
 package e2e
 
 import (
-	"log"
 	"testing"
 	"time"
 
+	"github.com/infracloudio/botkube/pkg/filterengine"
+	"github.com/infracloudio/botkube/pkg/log"
+
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/execute"
+
+	"github.com/stretchr/testify/require"
+
 	"github.com/slack-go/slack"
 
 	"github.com/infracloudio/botkube/pkg/bot"
```

Comment:
Lets group imports into 2 groups - 3rd party and internal. Please remove unnecessary new lines